framework/wazuh/core/cluster/master.py

```diff
@@ -440,7 +440,16 @@ def get_permission(self, sync_type: bytes) -> Tuple[bytes, bytes]:
         if sync_type == b'syn_i_w_m_p' and self.name not in self.server.integrity_already_executed:
             # Add the variable self.name to keep track of the number of integrity_checks per cycle
             self.server.integrity_already_executed.append(self.name)
-            permission = self.sync_integrity_free
+
+            # Reset integrity permissions if False for more than "max_locked_integrity_time" seconds
+            if not self.sync_integrity_free[0] and (datetime.utcnow() - self.sync_integrity_free[1]).total_seconds() > \
+                    self.cluster_items['intervals']['master']['max_locked_integrity_time']:
```

Comment:
We only need to use `datetime.utcnow()` when we set the flag value to False.

Reply:
Tuple immutability was the reason for reassigning everything each time. I have used lists to avoid this problem, as you suggested.